tests/tests.rs

```diff
@@ -622,6 +616,24 @@ fn test_default_builder_type() {
         }
     }
 
-    let outer = Outer::builder().a(3).b(5).build().outer();
+    impl From<Inner> for Outer {
+        fn from(value: Inner) -> Self {
+            Self(value)
+        }
+    }
+
+    let outer = Outer::builder().a(3).b(5).build();
     assert_eq!(outer, Outer(Inner { a: 3, b: 5 }));
 }
+
+#[test]
+fn test_into_set_generic() {
```

Comment:
@FlixCoder Just noticed this, but... how does this test the new feature exactly? `Into` into itself?

Reply:
It tests whether the generics are correctly set when generating the build function. The into works as per From trait and is basically negligble.

Reply:
But it basically converts from a type to itself - remove the `#[builder(build_method(into))]` and it'd still work.

I'm going to change it to use two type. Also, I'm going to make two versions - one that `impl From` and one that `impl Into`. I forgot that the standard Rust way is to check `Into` because there is blanket `impl` of `Into` if there is a `From`.

Reply:
The into specific type os in the test above with the outer already, but it is probably better to have a dedicated test right?

Reply:
Also the genericness requires From to be implemented, even though Into is used to convert, so it probably complain. I wonder if one can use `#name: Into<R>` instead to make it work.

Reply:
Yes, but it requires a `where` clause: https://github.com/idanarye/rust-typed-builder/commit/5cead25255bc7e9313cae40259b6e02172a3281a#diff-e8a945989e052d3a4f5c3366aa483e17330af013e0dacf70aed01f19da45237dR513

Reply:
Ah, nice catch, that is probably better